Application/EdFi.Ods.Standard/Standard/4.0.0/Artifacts/MsSql/Structure/Ods/1006-AddSQLTypes-BigInt.sql

```diff
@@ -0,0 +1,9 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TYPE dbo.BigIntTable AS TABLE
```

Comment:
Note that I'm introducing BigIntTable for DS 4.

Although not ideal, it doesn't represent a change from the current behavior; for example the next query is generated when we call a `/deletes` endpoint (using the `main` branch), notice that the parameters are bigint. 
With my change, the parameter would be a TVP of `BigIntTable`.

Is this acceptable? If not, I can make the code changes to check the DS version and downcast to int32 if < 5.0.0.

![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/assets/158314644/9b9c2e58-1765-4ac5-af21-1fe7ba49d692)

Reply:
I can't think of any big down sides to this approach (pun intended).